app/models/patient_specific_direction.rb

```diff
@@ -39,7 +39,11 @@ class PatientSpecificDirection < ApplicationRecord
   belongs_to :programme
   belongs_to :vaccine
 
-  validates :full_dose, inclusion: { in: [true, false] }
+  validates :full_dose,
```

Comment:
I'm wondering if we even need this column on the PSD? The result will be recorded on the vaccination record rather than the PSD I would imagine.